ui/server/src/test/scala/pl/touk/nussknacker/ui/component/DefaultComponentServiceSpec.scala

```diff
@@ -469,9 +506,120 @@ class DefaultComponentServiceSpec extends FlatSpec with Matchers with PatientSca
     )
 
     val wrongConfigurations = intercept[ComponentConfigurationException] {
-      DefaultComponentService(globalConfig, processingTypeDataProvider, fetchingProcessRepositoryMock, stubSubprocessRepository, categoryService)
+      DefaultComponentService(globalConfig, badProcessingTypeDataProvider, fetchingProcessRepositoryMock, stubSubprocessRepository, categoryService)
     }.wrongConfigurations
 
     wrongConfigurations should contain allElementsOf expectedWrongConfigurations
   }
+
+  it should "return component's processes for each user" in {
```

Comment:
I would say this test is too long and brittle... ProcessObjectsFinder has its tests... we don't need to replicate them here

Reply:
Changed, I was right. Now it looks better ;)